tests/test_ast.py

```diff
@@ -0,0 +1,40 @@
+from m2cgen import ast
+
+
+def test_count_exprs():
```

Comment:
Can you please add one really big expression which contains all expressions we have one or more times?